includes/class-wc-update-client.php

```diff
@@ -479,7 +479,7 @@ private function get_plugin_data($action, $plugin='') {
 
 		// Assemble args to post back to the Update Manager plugin.
 		$options = [
-			'user-agent' => 'ClassicPress/'.$cp_version.'; '.get_bloginfo('url'),
+			'user-agent' => 'WooCommerce/3.5.3 (compatible; ClassicCommerce/'.$cp_version.'); '.get_bloginfo('url'),
```

Comment:
This one should not be changed, the update manager server is expecting this to be in a specific format.